### ISSUE TITLE
/server and /servergui bring up no GUI

### ISSUE THREAD

**Issue by dirazi** [Matt-MX/ServerGUI#14]
Velocity up to date
Paper-Spigot (1.20.4)
Latest release of ServerGUI

When I execute the gui command, no message appears and no gui is opened.

Performing /servergui reload actually returns a reload confirmed message

Custom Shortcut commands for servers also appear to be working

**Comment by dirazi** [comment]
I'm a silly goober, this is a plugin for Velocity NOT paper.

User error :p

**Comment by superJK92iscool** [comment]
Same happens for me, except I am on Purpur (A Pufferfish fork, which is a Paper fork, which is a Spigot fork, in case anyone didn't know) for 1.18.2

**Comment by Shivter14** [comment]
Guys this is a Velocity plugin

**Comment by Matt-MX** [comment]
Plugin's hella outdated, might refactor it with my gui project when multiplatform impl is finished

**Comment by superJK92iscool** [comment]
> Guys this is a Velocity plugin

True, but the GUI doesn't open despite me having it installed on Velocity
